hoomd/hpmc/Moves.h

```diff
@@ -180,14 +149,14 @@ template<class RNG>
 inline vec3<Scalar> generatePositionInSphere(RNG& rng, vec3<Scalar> pos_sphere, Scalar R)
     {
     // draw a random vector in the excluded volume sphere of the colloid
```

Comment:
Not positive, but this file looks like it has some bits doing something similar to `SpherePointGenerator` if you can simplify the code that way.

Reply:
indeed. And `SpherePointGenerator` seems more robust by one `epsilon` :)